### PR TITLE
ArcListview: fix for selection highlight

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/318
[Problem] Arclistview: Highlight overlaps the scrollbar
[Solution] Position of Selection in DOM structure has been changed.

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: src/js/profile/wearable/widget/wearable/ArcListview.js [hunk 1]
```diff
@@ -1392,14 +1392,26 @@
...
 				if (!arcListviewElement) {
 					arcListviewElement = document.createElement("div");
 					arcListviewElement.classList.add(cssClass);
-					page.appendChild(arcListviewElement);
+					parentElement.appendChild(arcListviewElement);
+				}
+				return arcListviewElement;
+			};
+
+			prototype._buildArcListviewElementBefore = function (parentElement, cssClass) {
```
Comment (grzegorz-czajkowski) [inline src/js/profile/wearable/widget/wearable/ArcListview.js]: This method looks quite similar to _buildArcListviewElement. How about having one method where it's possible to specify insertion method, for example:

`_buildArcListviewElement(parentElement, cssClass, insertionOption) `

where insertionOption could be
```
{ insertBefore: true }
```

**Review by lslachciak — COMMENTED** [review]
works well, but please check @grzegorz-czajkowski comment

File: src/js/profile/wearable/widget/wearable/ArcListview.js (hunk 1 above)
Comment (TomaszLukawskiSam) [reply src/js/profile/wearable/widget/wearable/ArcListview.js]: You are right.

**Review by grzegorz-czajkowski — APPROVED** [review]
Looks fine.

**Review by lslachciak — APPROVED** [review]
works well

**Pull request merged**